src/openrct2/peep/Peep.cpp

```diff
@@ -1777,25 +1777,34 @@ rct_peep * peep_generate(sint32 x, sint32 y, sint32 z)
     peep->thoughts->type          = PEEP_THOUGHT_TYPE_NONE;
     peep->window_invalidate_flags = 0;
 
-    uint8 al = (scenario_rand() & 0x7) + 3;
-    uint8 ah = Math::Min(al, static_cast<uint8>(7)) - 3;
+    uint8 intensity_highest = (scenario_rand() & 0x7) + 3;
+    uint8 intensity_lowest = Math::Min(al, static_cast<uint8>(7)) - 3;
```

Comment:
`al` should be `intensity_highest` here. Please also rename the variables to `intensityHighest` and `intensityLowest`. We try to make the code more consistent by using camelCase for local variables and arguments (even though it's not explicitly stated on the coding-style page on the wiki yet).